activities/TamTamMicro.activity/piano.js

```diff
@@ -119,4 +120,13 @@ enyo.kind({
 	abort: function() {
 		this.endofsound();
 	},
+
+	destroy: function () {
+		var that = this;
+		window.clearInterval(that.job);
```

Comment:
Not sure to understand what is this `that.job` ?

Reply:
addressed in 
e281f29

Reply:
e281f29 my bad, was following a code sample and left that in by accident lol